exercises/chapter8/test/Main.purs

```diff
@@ -73,6 +74,19 @@ Note to reader: Delete this line to expand comment block -}
             $ filterM
                 onlyPositiveEvenIntegers
                 (2 : 3 : 4 : Nil)
+      suite "ST" do
+        suite "estimatePi" do
+          test "1000 terms of Gregory Series"
+            $ Assert.assert "Estimated value of pi not within threshold"
+            (estimatePi 1000 - pi < 0.002)
```

Comment:
Should we use absolute value here in case an off-by-one error makes the difference negative?

Reply:
Whoops! Yes.